### ISSUE TITLE
[Accessibility] [Command Bar]: NVDA doesn't announce the command bar as a menu with n items

### ISSUE THREAD

**Issue by erjohnms** [OfficeDev/office-ui-fabric-react#2924]
### Bug Report

- __Package version(s)__: 4.45.1000
- __Browser and OS versions__: chrome

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? N

Requested priority: Low

Products/sites affected: (if applicable)

#### Describe the issue:

When tabbing into a command bar, NVDA doesn't announce it as a menu with n items. The example docs work around this by changing the aria label of the first item to say, "New, use the left and right arrow keys to navigate" but I don't think consumers of the command bar should be expected to know that they need to create a more complex aria description for the first item because it will happen to be read first.

#### Actual behavior:

NVDA reads the first item in the command bar when tabbed into

#### Expected behavior:

It should read something like, "menu item 1 of 8"

### If applicable, please provide a codepen repro:

You can use a screen reader on the office fabric command bar docs:
https://dev.office.com/fabric#/components/commandbar
and see that it works around this problem with a specialized aria label on the first item which is a weird detail that every consumer of command bar will need to know and implement.
